bears/c_languages/CPPLintBear.py

```diff
@@ -26,12 +32,17 @@ class CPPLintBear:
     def create_arguments(filename, file, config_file,
                          max_line_length: int=79,
                          cpplint_ignore: typed_list(str)=(),
-                         cpplint_include: typed_list(str)=()):
+                         cpplint_include: typed_list(str)=(),
+                         indent_size: int=2,
```

Comment:
Do we want these as defaults or do we want the user to fail without any given argument. With a default value, users might think 4 just works. But I guess they should read the docs and then they'd see.

Reply:
Case A: Fails without any given argument. Then maybe they set this argument to 4, which fails again. Then they read the docs and set it to 2.
Case B: Works by default. If user wants to override, docs clearly call out the permitted value.
Thoughts?